tests/darwin/dataset/identifier_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from darwin.dataset.identifier import DatasetIdentifier
+
+
+class TestDatasetIdentifier:
```

Comment:
Instead of defining a new class I have installed ```pytest-describe``` and I am using describe blocks in my tests. Perhaps we should have a discussion to see which convention we ought to adopt.

Reply:
Interesting, let me have a look

Reply:
Adopted because:
- classes is imho a terrible means of structuring/organizing tests
- `pytest-describe` allows you to get rid of the `test_` naming nonsense